tests/ablateLibrary/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_executable(libraryTests "")
+add_executable(libraryTests "" eos/stiffenedGasTests.cpp)
```

Comment:
Do you mind reverting this file and added stiffenedGasTests.cpp to the cmakelist file in the ablate/tests/ablateLibrary/eos/CMakeLists.txt file?